summarizer_server/server.py

```diff
@@ -13,24 +18,25 @@
 from text_rank import TextRank
 
 debug = os.environ.get("DEBUG", "false").lower() == "true"
+ENV = os.environ.get("ENV", "dev")
+DD_API_URL = "https://api.datadoghq.com/api/v1/"
 
 log = logging.getLogger("summarizer_server")
 
 app = Flask(__name__)
-app.config.from_object(Settings)
+textrank = TextRank()
+textrank.setup()
```

Comment:
I remember there being a reason that we had `textrank.setup()` in the `before_first_request` block. Can you test to make sure that this works?

Reply:
Ya i've tested it locally, I think we should be good.